### PR TITLE
Fixes/cleanups for SSH key/systemd unit local file embedding

### PR DESCRIPTION
**Pull request by bgilbert**
Address various small items.  Follows up on #289.

### CODE REVIEW

File: base/v0_5_exp/translate_test.go [hunk 1]
```diff
@@ -1935,257 +1935,158 @@ func TestTranslateSSHAuthorizedKey(t *testing.T) {
...
 			sshKeyDir,
 		},
 		{
-			"valid ssh_keys_local with multiple keys per file",
+			"valid local keys with multiple keys per file",
 			PasswdUser{SSHAuthorizedKeysLocal: []string{sshKeyMultipleKeysFileName}},
 			types.PasswdUser{SSHAuthorizedKeys: []types.SSHAuthorizedKey{
 				types.SSHAuthorizedKey(sshKey2),
 				types.SSHAuthorizedKey("#comment"),
-				types.SSHAuthorizedKey(""),
```
Comment (travier) [inline base/v0_5_exp/translate_test.go:1977]: Aren't those empty lines here to make sure empty lines are correctly ignored?

**Comment by travier** [comment]
Otherwise LGTM.

File: base/v0_5_exp/translate_test.go (hunk 1 above)
Comment (travier) [reply base/v0_5_exp/translate_test.go:1977]: Maybe we should have a dedicated test for that case instead?

**Review by travier — APPROVED** [review]

**Review by prestist — APPROVED** [review]
Wow, nice fixup! 
LGTM

**Pull request merged**